preprocessor.py

```diff
@@ -75,32 +93,64 @@ def tag_usernames(text):
 
 
 def preprocess(
-    df, emojis, emoticons, urls, html_tags, acronyms, spelling, usernames,
+    in_filename,
+    out_filename,
+    emojis,
+    emoticons,
+    urls,
+    html_tags,
+    acronyms,
+    spelling,
+    usernames,
+    max_workers,
+    chunksize,
 ):
-    if emojis:
-        print("Removing emojis...")
-        df["text"] = df["text"].progress_apply(lambda text: remove_emojis(text))
-    if emoticons:
-        print("Removing emoticons...")
-        df["text"] = df["text"].progress_apply(lambda text: remove_emoticons(text))

```

Comment:
Moved this two operations down because the emoticons was affecting URLs.
The emoticon :/ was being removed, which changed the https:// to https/